src/VirtualClient/VirtualClient.Main/profiles/PERF-GPU-MLPERF-NVIDIA-H100.json

```diff
@@ -0,0 +1,93 @@
+{
+    "Description": "MLPerf GPU Performance Workload",
+    "Metadata": {
+        "RecommendedMinimumExecutionTime": "10:00:00",
+        "SupportedPlatforms": "linux-x64",
+        "SupportedOperatingSystems": "Ubuntu",
+        "SpecialRequirements": "This is an Nvidia GPU-specialized workload. It depends upon the system having an Nvidia GPU card/chip.",
+        "GPU": "H100"
+    },
+    "Parameters": {
+        "Username": "",
+        "DiskFilter": "SizeGreaterThan:1000gb",
+        "RepoUri": "https://github.com/mlcommons/inference_results_v3.1.git",
+        "LinuxCudaVersion": "12.2",
+        "LinuxDriverVersion": "535",
+        "LinuxLocalRunFile": "https://developer.download.nvidia.com/compute/cuda/12.2.0/local_installers/cuda_12.2.0_535.54.03_linux.run",
+        "RequireCustomSystemSupport": false,
+        "Benchmarks": "bert,rnnt,retinanet"
+    },
+    "Actions": [
+        {
+            "Type": "MLPerfExecutor",
+            "Parameters": {
+                "Scenario": "inferencemodels-bert",
+                "Model": "bert",
+                "Username": "$.Parameters.Username",
+                "DiskFilter": "$.Parameters.DiskFilter",
+                "RequireCustomSystemSupport": "$.Parameters.RequireCustomSystemSupport",
+                "Benchmarks": "$.Parameters.Benchmarks"
```

Comment:
I don't understand here why you have separate scenario names but they are running the same benchmark

Reply:
Model parameter determines which benchmark will be run, this benchmarks parameter is the list of all benchmarks that will be run as part of complete profile, reason being we prepare the setup by downloading all the required datasets and then prebuild for those required benchmarks and then each model is run

Reply:
Can we prepare the dataset only for the current defined workload? Instead of everything upfront?